autobot/telegram/objects/inlinekeyboard.py

```diff
@@ -1,5 +1,6 @@
 from .base import BaseObject
-
+from . import InlineKeyboardButton, WebAppInfo, LoginUrl
+from typing import List, Optional
```

Comment:
type hint `List` is deprecated so use `list` instead as mentioned by @ht-thomas

Reply:
okay but then, I want to type hint that the property expects a list of `InlineKeyboardButton` (s). using just `list` is not enough to achieve what I want. I can not do `list[InlineKeyboardButton]` because `list` is not subscriptable . 
So how do go about this?

Reply:
```python
class Foo:
    def __init__(self, a: int) -> None:
        self.a = a

foo_list: list[Foo] = [Foo(1), Foo(2)]
```
Here `foo_list` is a list of `Foo` objects thus it's achieavable

Reply:
I was testing the implementation on python 3.8.2 and it was giving me an error. 
It works on v.3.9^

Let me correct it. Thank you.

Reply:
I was hoping we were going to use Python 3.10 as our minimum supported version, that's why all type hints were targetting that.

Reply:
Alright then. I have changed the typing as required.